app-admin/opendoas/spec

```diff
@@ -0,0 +1,3 @@
+VER=6.8.2
+SRCS="tbl::https://github.com/Duncaen/OpenDoas/archive/refs/tags/v$VER.tar.gz"
```

Comment:
```suggestion
SRCS="git::commit=tags/v$VER::https://github.com/Duncaen/OpenDoas"
```

Reply:
It this really needed?
`git grep 'archive/$VER' | wc -l` gives me 211.

Reply:
GitHub and GitLab generate stable tarballs, dont think it is a problem.

Reply:
see [package styling manual chapter 3.2](https://wiki.aosc.io/developer/packaging/package-styling-manual/#source-variables)